modules/network/nsg-bastion.tf

```diff
@@ -2,26 +2,27 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
 locals {
```

Comment:
Please document the changes here to facilitate the review

Reply:
Done